.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+<!-- Read comments, before committing pull request read checklist again
+
+# Checklist:
+
+- I have performed a self-review of my own code,
+- I have commented my code, particularly in hard-to-understand areas,
+- Title include "WIP" if work is in progress.
+
+-->
+
+Resolves #01 <!-- issue number -->
+
+### Description
+
+<!--
+
+- Please include a summary of the change and which issue is fixed.
+- List any dependencies that are required for this change. (gems, js libraries, etc.)
+
+-->
+
+### Type of change
+
+<!-- Please delete options that are not relevant. -->
+
+- [ ] Bug fix
+- [ ] New feature
+- [ ] This change requires a documentation update
```

Comment:
Lets update this to keep consistency between frontend and backend

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update